### PR TITLE
Benchmarks: Fix Bug - Fix wrong parameters for gpu-sm-copy-bw in configuration examples

### PR DESCRIPTION
**Pull request by yzygitzh**
**Description**
This commit fixes wrong parameters for gpu-sm-copy-bw call in configuration examples.

### CODE REVIEW

File: superbench/config/azure_ndv4.yaml [hunk 1]
```diff
@@ -35,6 +35,25 @@ superbench:
...
+    disk-benchmark:
+      enable: false
+      modes:
+        - name: local
+          proc_num: 1
+          parallel: no
+      parameters:
+        block_devices: []
+    gpu-sm-copy-bw:
+      enable: false
```
Comment (guoshzhao) [inline superbench/config/azure_ndv4.yaml:47]: For `gpu-sm-copy-bw`, I think we can enable it by default if it works fine.

**Review by guoshzhao — APPROVED** [review]

File: superbench/config/azure_ndv4.yaml (hunk 1 above)
Comment (yzygitzh) [reply superbench/config/azure_ndv4.yaml:47]: It takes around 5min to finish on a 4 NUMA 8 GPU machine. We can enable it by default if it's acceptable. What do you think?

Comment (cp5555) [reply superbench/config/azure_ndv4.yaml:47]: I think Guoshuai mean that you should enable it. Otherwise, we should remove it.

**Comment by yukirora** [comment]
why does this pr merge to main?

File: superbench/config/azure_ndv4.yaml (hunk 1 above)
Comment (yzygitzh) [reply superbench/config/azure_ndv4.yaml:47]: Fixed.

**Comment by yzygitzh** [comment]
> why does this pr merge to main?

Do you have suggestion for merging target branch?

**Comment by yukirora** [comment]
> > why does this pr merge to main?
> 
> Do you have suggestion for merging target branch?

I think it should be merged to release/0.3

**Comment by yzygitzh** [comment]
> > > why does this pr merge to main?
> > 
> > 
> > Do you have suggestion for merging target branch?
> 
> I think it should be merged to release/0.3

I think we should first merge to dev branch and then backport to 0.3 if necessary.

**Review by cp5555 — APPROVED** [review]

**Pull request merged**